templates/index.html

```diff
@@ -1,15 +1,15 @@
 <title>Community website</title>
```

Comment:
why no `<html lang="en">` required?

Reply:
No idea, why it was not added at the first place. I will add it 👍